python/lsst/analysis/tools/actions/plot/propertyMapPlot.py

```diff
@@ -362,35 +366,56 @@ def makePlot(
             plotInfo["plotName"] = self.plotName
 
         # Plotting customization.
+        colorbarLocation = "right"
         colorbarTickLabelSize = 14
         colorBarAspect = 16
+        colorBarFraction = 0.07  # w.r.t. the axes size.
+        colorBarPad = 0  # w.r.t. the axes size.
         rcparams = {
-            "axes.labelsize": 18,
+            "axes.labelsize": 22 if plotConfig.publicationStyle else 19,
             "axes.linewidth": 1.8,
-            "xtick.labelsize": 13,
-            "ytick.labelsize": 13,
+            "xtick.labelsize": 15 if plotConfig.publicationStyle else 14,
+            "ytick.labelsize": 15 if plotConfig.publicationStyle else 14,
         }
-        zoomFactors = plotConfig.zoomFactors
 
-        # Muted green for the full map, and muted red and blue for the two
-        # zoomed-in maps.
-        histColors = ["#265D40", "#8B0000", "#00008B"]
         if plotConfig.publicationStyle:
-            histColors = ["#440154", "#31688e", "#35b779"]
+            zoomFactors = []
+            _LOG.info(
+                "Zoom factors are not used in publication-style plots. "
+                "Only the full-tract map will be plotted."
+            )
+            histColors = ["#440154"]
+            rasterized = False
+            labelpad = 14
+        else:
+            zoomFactors = plotConfig.zoomFactors
+            # Muted green for the full map, and muted red and blue for the two
+            # zoomed-in maps.
+            histColors = ["#265D40", "#8B0000", "#00008B"]
```

Comment:
Can we use colours from the colourmap palette? Maybe the two ends and the middle? https://waldyrious.net/viridis-palette-generator/ can pull out the hexcodes.